requirements.txt

```
@@ -0,0 +1,2 @@
+flask
+Flask-RESTful
```

```suggestion
flask==2.2.1
Flask-RESTful==0.3.9
```

Em se tratando de dependências, é importante especificar a versão de cada uma. Se não for não forem especificadas, a instalação das dependências não é determinística, ou seja, pode ser que você instale as dependências numa versão hoje e outra pessoa instale outras versões delas no futuro mesmo se vocês dois rodaram `pip install -r requirements.txt`.

As versões sugeridas são daqui:

- https://pypi.org/project/Flask-RESTful/
- https://pypi.org/project/Flask/